src/components/HelloWorld.vue

```diff
@@ -2,30 +2,29 @@
   <div class="hello">
     <h1>{{ msg }}</h1>
     <p>
-      For a guide and recipes on how to configure / customize this project,<br>
+      For guide and recipes on how to configure / customize this project,<br>
```

Comment:
The first one was proper grammar, but we'll be deleting this anyway so it doesn't matter